src/Media_Command.php

```diff
@@ -488,10 +494,34 @@ public function image_size( $args, $assoc_args ) {
 				'width'     => '',
 				'height'    => '',
 				'crop'      => 'N/A',
+				'ratio' 		=> 'N/A',
 		) );
 		WP_CLI\Utils\format_items( $assoc_args['format'], $sizes, explode( ',', $assoc_args['fields'] ) );
 	}
 
+	private function get_ratio( $width, $height ) {
+		if ( $height == 0 ) {
+			$image_ratio 	= "0:{$width}";
+		} elseif ( $width == 0 ) {
+			$image_ratio 	= "{$height}:0";
+		} else {
+	    $gcd          = $this->gcd( $width, $height );
+			$width_ratio  = $width / $gcd;
+			$height_ratio = $height / $gcd;
+			$image_ratio  = "{$width_ratio}:{$height_ratio}";
+		}
+		return $image_ratio;
```

Comment:
You should prefer early returns instead of else clauses.
Something like this:
```
if ( $height == 0 ) {
   return "0:{$width}";
}

if ( $width == 0 ) {
   return "{$height}:0";
}

$gcd          = $this->gcd( $width, $height );
$width_ratio  = $width / $gcd;
$height_ratio = $height / $gcd;
return "{$width_ratio}:{$height_ratio}";

Reply:
Thanks @schlessera for the quick review! I'll take a look tomorrow and adjust :)